src/quark2_adapter/adapters.py

```diff
@@ -0,0 +1,208 @@
+from copy import deepcopy
+import json
+from time import time
+import logging
+
+from modules.Core import Core
+
+from modules.applications.Application import Application as Application_NEW
+from quark2_adapter.legacy_classes.Application import Application as Application_OLD
+from modules.applications.Mapping import Mapping as Mapping_NEW
+from quark2_adapter.legacy_classes.Mapping import Mapping as Mapping_OLD
+from modules.solvers.Solver import Solver as Solver_NEW
+from quark2_adapter.legacy_classes.Solver import Solver as Solver_OLD
+from modules.devices.Device import Device as Device_NEW
+from quark2_adapter.legacy_classes.Device import Device as Device_OLD
+
+
+WARNING_MSG = 'Class "%s" is inheriting from depricated base class. Please refactor your class.'
+
+
+class ApplicationAdapter(Application_NEW, Application_OLD):
+    def __init__(self, application_name, *args, **kwargs):
```

Comment:
Please add a doc string for each function of all classes

Reply:
done together with some other minor changes